robot/Cumulus/resources/locators_49.py

```diff
@@ -241,29 +241,37 @@
         'select_related':'//select[@id = "DomainEnumOrId"]',
         'select_related_option':'//select[@id = "DomainEnumOrId"]/option[@value="{}"]',
         'search_result': '//tbody/tr/td/a/span[contains(text(),"{}")]',
+        'formula_txtarea': '//textarea[@id = "{}"]',
+        'input_validation': '//input[@title = "Check Syntax"]',
+        'object_result': '//th/a[text()="{}"]',
+        'sidenav_option':'//a[text()="{}"]'
         },
     'custom_settings':{
         'subtree':'//a/mark[text()="{}"]',
         'link':"//table[@class='list']/tbody/tr[./th/a[text()='{}']]/td/a[text()='{}']",
-        'cbx_status':'//table[@class="detailList"]/tbody/tr/th[./span[text()="{}"]]/following-sibling::td//img[@title="{}"]',
+        'cbx_status':'//table[@class="detailList"]/tbody/tr/th[./span[text()="{}"]]/following-sibling::td//img[@title="{}"]'
         },
     'adv_mappings':{
         'dropdown':"//tr[.//*[text()='{}']]/td[.//span[text()='Show actions']]//button",
         'modal_open':'//div[contains(@class,"slds-backdrop_open")]',
         'field_mapping':'//input[@name="{}"]',
         'combobox':'//div[contains(@class,"slds-is-open") and @role="combobox"]',
         'footer-btn':"//footer[@class='slds-modal__footer']/button[text()='{}']", 
-        'field-label':'//lightning-primitive-cell-factory//*[text()="{}"]',
+        'field-label':'//lightning-primitive-cell-factory//*[text()="{}"]'
         },
     'modal-form':{
         'label':'//div[./*/*[text()="{}"]]',
         },
+	# Customizable rollups related element locators
     'crlps':{
         'select_locator': "//select[@name ='{}']",
         'success_toast': "//div[@class='{}']",
         'active_setting_record': "//td[@data-label='Active']/following::tr/th//span/div//lightning-button/button[text()='{}']",
         'rollup_progress_notification': "//div/h2[contains(text(),'{}')]",
         'rollup_options':"//tr[./th//button[text()='{}']]/td//button",
+        'button': '//button[text() = "{}"]',
+        'modal-button': '//footer/button[text()="Save"]',
+        'filter_group': "//lightning-button/button[text() = '{}']",
```

Comment:
curious, cant u use same locator for 272 and 274 lines? Also we already have a 'button-with-text':'//button[contains(text(),"{}")]', will this not work?

Reply:
addressed